docs/articles/new-expensify/billing-and-subscriptions/Billing-Overview.md

```diff
@@ -1,52 +1,90 @@
+**Settings > Account > Subscription > Billing History**
 
-- **Collect Plan:** $5 per active member per month 
-- **Control Plan:** $9 per active member per month
+**Note:** We recommend having just one Billing Owner per Group Workspace to simplify account management.
 
-**You also get cash back!** Earn 1% cash back on all Expensify Card purchases or 2% if card spending reaches $250,000 or more monthly (for US purchases only). Cash back first applies to your Expensify bill, with any remainder deposited directly into your bank account.
+---
+
+# Save Money on Expensify
+
+## Best Discount: Annual + Expensify Card
+
+Pair an **annual subscription** with the **Expensify Card** to get the lowest pricing.  
+If at least **50%** of your total settled US spend in a month is on the Expensify Card, you'll qualify for the **best price**:
+
+- **Collect Plan:** $5 per active member/month  
+- **Control Plan:** $9 per active member/month
+
+### Additional Cash Back
+- **1% cash back** on all Expensify Card purchases  
+- **2% cash back** if total US spend on the card is $250,000+ per month
+
+Cash back is first applied to your Expensify bill. Any remaining balance is deposited into your bank account.
 
-Use Expensify’s [savings calculator](https://use.expensify.com/resource-center/tools/savings-calculator) to see your potential savings with the Expensify Card. 
+👉 Use our [savings calculator](https://use.expensify.com/resource-center/tools/savings-calculator) to see how much you could save.
 
-## Annual Subscription vs Pay-per-use
+---
+
+## Annual Subscription vs. Pay-per-use
+
+### Annual Subscription
+
+Get **50% off** with an annual commitment:
 
-**Annual Subscription**
+| Plan        | Annual Rate (per active member/month) |
+|-------------|----------------------------------------|
+| Collect     | $10                                    |
+| Control     | $18                                    |
 
-You get a 50% discount with an annual subscription:
+If you go over your subscription size, extra members are charged at the **pay-per-use rate** for that month.
 
-- **Collect Plan:** $10 per active member per month 
-- **Control Plan:** $18 per active member per month
-  
-If your active members exceed your subscription size, additional members are billed at the pay-per-use rate for that month. You can increase your subscription size at any time (extending your annual term) but can only reduce it once your current subscription period ends.
+You can increase your subscription size anytime (which extends your annual term). You can **only decrease** it when your current term ends.
 
-**Pay-per-use**
+### Pay-per-use (aka Pay-as-you-go)
 
-Rates for pay-per-use plans are applied at full price:
+This plan charges full rates and is billed monthly:
+
+| Plan        | Pay-per-use Rate (per active member/month)         |
+|-------------|-----------------------------------------------------|
+| Collect     | $5 (intro rate for new users; conditions apply)     |
```

Comment:
Brought this back for discussion here: https://expensify.slack.com/archives/C07HPDRELLD/p1744743349908519?thread_ts=1744293834.075229&cid=C07HPDRELLD